include/zephyr/net/ethernet.h

```diff
@@ -196,6 +196,15 @@ enum ethernet_hw_caps {
 
 	/** TX-Injection supported */
 	ETHERNET_TXINJECTION_MODE	= BIT(20),
+
+	/**
+	 * Hint to upper layers that this Ethernet driver would like to have
```

Comment:
This is not really a hardware capability.

Reply:
True, but it is a driver capability, we have already several of them in the capability list. There does not seem to be easy way the driver to say that it could support this.